app/controllers/application_controller.rb

```diff
@@ -1,2 +1,17 @@
 class ApplicationController < ActionController::Base
+  before_action :authorize
+
+  def authorize
+    unless current_user
+      redirect_to login_path, notice: t('notice.session.login_before_continue')
```

Comment:
create file, config/locales/controllers/application/en.yml
and declare 
en:
  application: 
    login: "Please Log in"